docs/server-setup.md

```diff
@@ -13,11 +13,12 @@ I use and like [DigitalOcean](https://m.do.co/c/d371ed7f99af) (referral link) fo
 
 Make sure to choose the `NodeJS x.x.x on 16.04` (where `x.x.x` is the highest version number you can find) one-click app when creating the droplet to get the server fully setup with Node and Ubuntu.
 
-## Start and deamonize the app
+<a id="start-and-deamonize-the-app"></a>
```

Comment:
Since there was a typo in a section header, I've added an anchor to make sure that old links to the section still work. Let me know what you think about that!